src/quantization.cpp

```diff
@@ -103,5 +123,80 @@ void quantize(program& prog, const std::vector<std::string>& ins_names)
 
 void quantize(program& prog) { quantize(prog, {"all"}); }
 
+// For the input of each input argument, we need to insert a
+// capture operator to compute the scale and shift
+void capture_arguments(program& prog,
+                       const std::vector<std::string>& ins_names,
+                       std::function<void(std::size_t, std::vector<argument>)> func)
+{
+
+    size_t num_quant_params = 0;
+    // the int8 quantization only support dot and convolution
+    std::vector<std::string> op_names = {"dot", "convolution"};
```

Comment:
I think using a std::set would be better because then you could do something like if(not op_names == ins_names) MIGRAPHX_THROW(...) instead of doing find() and all_of()

Reply:
The inputs ins_name could be a subset of op_name, they do not have to be the same.

Reply:
you could use std::includes(op_names.begin(), op_names.end(), ins_names.begin(), ins_names.end()) to check for a subset

Reply:
Using `std::includes` is a good suggestion, but they must be sorted first, which can be done by putting it in a `std::set` or calling `std::sort` on the vector.